tests/test_obstacle_map.py

```diff
@@ -0,0 +1,107 @@
+from SDA import *
+import unittest
+import numpy 
+
+class Test_Obstacle_map(unittest.TestCase):
```

Comment:
The naming convention for unit testing classes is ```ThingToTestTestCase``` so for this set of unit tests you should do ```ObstacleMapTestCase```